wait_for_jerk: false
window_length: 0.75      # second
excite_threshold: 0.25